maintainers/maintainer-list.nix

```diff
@@ -11162,6 +11162,12 @@
     githubId = 26458780;
     name = "Matthew Croughan";
   };
+  mdsp = {
```

Comment:
Please put this into a separate commit with the message ``maintainers: add mdsp`` and sort it alphabetically.